### PR TITLE
Multi checkbox

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Add multi checkbox component

**Related issue(s)**

### CODE REVIEW

File: src/components/Extensibility/components-form/MultiCheckbox.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  storeKeys,
+  required,
+  resource,
+  compact,
+  placeholder,
+}) {
+  const { tFromStoreKeys, t: tExt, exists } = useGetTranslation();
+  const schemaPlaceholder = schema.get('placeholder');
+  const readOnly = schema.get('readOnly') ?? false;
+  const value = getValue(storeKeys, resource);
```
Comment (qbalukom) [inline src/components/Extensibility/components-form/MultiCheckbox.js:30]: We already have a workaround for this.
```suggestion
  value = getObjectValueWorkaround(schema, resource, storeKeys, value);
```
(see for example `src/components/Extensibility/components-form/ResourceRefRenderer.js`)

File: src/components/Extensibility/components-form/MultiCheckbox.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        };
+      });
+      console.log('!!!displayOptions', displayOptions);
+      return {
+        input: Inputs.Checkboxes,
+        options: displayOptions,
+      };
+    } else {
+      return {
+        input: Inputs.Switch,
```
Comment (qbalukom) [inline src/components/Extensibility/components-form/MultiCheckbox.js:82]: do we really want the fallback to switch? i would expect this component to simply require the `options` parameter. After all it's supposed to generate a list of values, when a switch generates a boolean, which is a completely different value type.

File: docs/extensibility/form-section.md [hunk 3]
```diff
@@ -333,6 +334,37 @@ Resource widgets render a dropdown list of specified resources and store the sel
 
 <img src="./assets/form-widgets/Resource.png" alt="Example of a Resource widget" style="border: 1px solid #D2D5D9">
 
+### MultiCheckbox
+
+MultiCheckbox widget render checkboxes that will be put saved into one path as array of strings.
```
Comment (IwonaLanger) [inline docs/extensibility/form-section.md:339]:
```suggestion
MultiCheckbox widgets render checkboxes that are saved into one path as an array of strings.
```

File: docs/extensibility/form-section.md [hunk 4]
```diff
@@ -333,6 +334,37 @@ Resource widgets render a dropdown list of specified resources and store the sel
...
+
+#### Widget-specific parameters
+
+- **options[]** - _[required]_ an array of objects to generate the checkboxes.
+  - **key** - _[required]_ a value used to generate checkboxes. Will be set if checkbox is selected. This `key` together with `path` defined for MultiCheckbox (`{path}.{key}`) can be used for **translation**.
```
Comment (IwonaLanger) [inline docs/extensibility/form-section.md:344]:
```suggestion
  - **key** - _[required]_ a value used to generate checkboxes. It is set if a checkbox is selected. This `key`, together with the `path` defined for MultiCheckbox (`{path}.{key}`), can be used for **translation**.
```

**Review by IwonaLanger — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**